app/views/layouts/blacklight/base.html.erb

```diff
@@ -28,7 +28,7 @@
   <main id="main-container" class="<%= container_classes %>" role="main" aria-label="<%= t('blacklight.main.aria.main_container') %>">
     <%= content_for(:container_header) %>
 
-    <%= render partial: 'shared/flash_msg', layout: 'shared/flash_messages' %>
+    <%= render partial: 'shared/flash_msg', layout: 'shared/flash_messages' unless params[:id] && current_page?(solr_document_path)%>
```

Comment:
please revert this.  This box is not for all the messages, only the message about the workflow error state.